pkg/testskeleton/testskeleton.go

```diff
@@ -107,10 +85,11 @@ func GenerateAzureRandomNames() AzureRandomNames {
 	gid := idSliced[0:2]
 	storageId := idSliced[3:5]
 
-	names := AzureRandomNames{
-		NamePrefix:         fmt.Sprintf("ghci%s%s-", prid, prefixId),
-		ResourceGroupName:  strings.Join(gid, ""),
-		StorageAccountName: fmt.Sprintf("ghci%s", strings.Join(storageId, "")),
+	names := TerraformVarsInfo{
+		NamePrefix:              fmt.Sprintf("%s-%s-", prid, prefixId),
+		AzureResourceGroupName:  strings.Join(gid, ""),
+		AzureStorageAccountName: fmt.Sprintf("ghci%s", strings.Join(storageId, "")),
+		GCPProjectID:            project_id,
```

Comment:
If in line 79 we are going to use camel case, then here we also need to update it into:

```suggestion
		GoogleProjectId:            projectId,
```

Additionally I replaced `GCP` by `Google`.

Do you accept that approach ?